pkgs/by-name/ey/eyewitness/package.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python311Packages
+, cmake
+, xvfb-run
+, firefox-esr
+, geckodriver
+}:
+
+stdenv.mkDerivation rec {
+  pname = "eye-witness";
+  version = "20230525.1";
+
+  src = fetchFromGitHub {
+    owner = "redsiege";
+    repo = "EyeWitness";
+    rev = "v${version}";
+    hash = "sha256-nSPpPbwqagc5EadQ4AHgLhjQ0kDjmbdcwE/PL5FDL4I=";
+  };
+
+  buildsystems = with python311Packages; [ setuptools ];
+
+  dependencies = with python311Packages; [
+    selenium
+    fuzzywuzzy
+    pyvirtualdisplay
+    pylev
+    netaddr
+    pydevtool
+  ] ++ [
+    firefox-esr
+    xvfb-run
+    geckodriver
+  ];
+
+  doCheck = false;
+
+  installPhase = ''
+    mkdir -p $out/{bin,share/eyewitness}
+    #mkdir -p $out/share/eyewitness
+    cp -R * $out/share/eyewitness
+    cp -R ${python311Packages.selenium}/lib/python3.11/site-packages/selenium $out/share/eyewitness/Python/modules
+    sed -i "/from ssl import SSLError as sslerr/a \sys.path.insert(1, '$out/share/eyewitness/Python/modules')" $out/share/eyewitness/Python/modules/selenium_module.py
+    bin=$out/bin/eyewitness
+    cat > $bin << EOF
+    #!/usr/bin/env sh
+    exec python3.11 $out/share/eyewitness/Python/EyeWitness.py
+    EOF
```

Comment:
Why not use makeWrapper?

Reply:
All problems solved now can you check for any more adjustments?